🪰Fly Deployがライブラリインストール失敗で落ちる

## 落ちたActions

https://github.com/momocus/sakazuki/actions/runs/9233787614/job/25406635668

## ログ

```console
#16 [build_deps 1/1] RUN --mount=type=cache,id=dev-apt-cache,sharing=locked,target=/var/cache/apt     --mount=type=cache,id=dev-apt-lib,sharing=locked,target=/var/lib/apt     apt-get update -qq &&     apt-get install --no-install-recommends -y build-essential=12.9 libpq-dev=13.*     && rm -rf /var/lib/apt/lists /var/cache/apt/archives
#16 1.075 Reading package lists...
#16 1.630 Building dependency tree...
#16 1.817 Package libpq-dev is not available, but is referred to by another package.
#16 1.817 This may mean that the package is missing, has been obsoleted, or
#16 1.817 is only available from another source
#16 1.817 
#16 1.821 E: Version '13.*' for 'libpq-dev' was not found
#16 ERROR: process "/bin/bash -o pipefail -c apt-get update -qq &&     apt-get install --no-install-recommends -y ${BUILD_PACKAGES}     && rm -rf /var/lib/apt/lists /var/cache/apt/archives" did not complete successfully: exit code: 100
------
 > [build_deps 1/1] RUN --mount=type=cache,id=dev-apt-cache,sharing=locked,target=/var/cache/apt     --mount=type=cache,id=dev-apt-lib,sharing=locked,target=/var/lib/apt     apt-get update -qq &&     apt-get install --no-install-recommends -y build-essential=12.9 libpq-dev=13.*     && rm -rf /var/lib/apt/lists /var/cache/apt/archives:
1.075 Reading package lists...
1.630 Building dependency tree...
1.817 Package libpq-dev is not available, but is referred to by another package.
1.817 This may mean that the package is missing, has been obsoleted, or
1.817 is only available from another source
1.817 
1.821 E: Version '13.*' for 'libpq-dev' was not found
------
Error: failed to fetch an image or build from source: error building: failed to solve: process "/bin/bash -o pipefail -c apt-get update -qq &&     apt-get install --no-install-recommends -y ${BUILD_PACKAGES}     && rm -rf /var/lib/apt/lists /var/cache/apt/archives" did not complete successfully: exit code: 100
Error: Process completed with exit code 1.
```

--- a/fly.Dockerfile
+++ b/fly.Dockerfile
@@ -19,7 +19,8 @@
 
 ARG RUBY_VERSION=3.3.1
 ARG VARIANT=jemalloc-slim
-FROM quay.io/evl.ms/fullstaq-ruby:${RUBY_VERSION}-${VARIANT} as base
+ARG OS_VERSION=bullseye
+FROM quay.io/evl.ms/fullstaq-ruby:${RUBY_VERSION}-${OS_VERSION}-${VARIANT} as base
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 

